app/src/main/res/layout/contacts_list_item.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:focusable="false"
```

Comment:
why do you use focusable and focusableInTouchMode ? These are likely used when you have a focusable control that will steal the click event